### ISSUE TITLE
Not able to install the application

### ISSUE THREAD

**Issue by swathi11111** [abdeladim-s/subsai#142]
I tried all the required dependent libraries installations and then cloned the SubSai repository. When I try to run the command "subsai-webui" on command prompt it says this batch file does not exist. I don't see any file named with this being cloned on my system. Can someone please help me with the installation, I really need this tool.

**Comment by abdeladim-s** [comment]
@swathi11111, Have you installed the package using `pip install git+https://github.com/abdeladim-s/subsai` ? 
if it went okey, the command `subsai-webui` will be available on your environment, you don't need to clone anything!

**Comment by swathi11111** [comment]
I have done the same, can you help me with the installation by being on a google meet call if possible?

**Comment by abdeladim-s** [comment]
I wish I can but I am very busy! 
However there is an easier way to install the package if you are stuck installing it with the first method, you can use Docker, this will take some time to build and also some storage space, but it's going to work.

First download it from [here](https://docs.docker.com/desktop/install/windows-install/), there are many resources out there to get you started if you are not familiar with it.
Then follow the instructions from the [Readme](https://github.com/abdeladim-s/subsai#docker), 

Let me know if you find any issues.

**Comment by swathi11111** [comment]
@CorradoF, I have done that too,  added paths of python, pip and rustc. I was even able to get the right versions printed on command prompt of each package using python --version, pip --version and rustc --version. Even after doing this, am not able to install the application.

**Comment by abdeladim-s** [comment]
@swathi11111, What Python version are you using ? 
Python 3.12 does not work well! Please use Python 3.11 or 3.10! 
Hopefully this will resolve your issue!

**Comment by ChiNoel-osu** [comment]
> [@swathi11111](https://github.com/swathi11111), What Python version are you using ? Python 3.12 does not work well! Please use Python 3.11 or 3.10! Hopefully this will resolve your issue!

aw man we should put this in readme, took me like an hour of troubleshooting to find out

**Comment by absadiki** [comment]
@ChiNoel-osu, True, I’ve added it to the README. thanks for bringing this to my attention!
